pyproject.toml

```
@@ -33,6 +51,14 @@ classifiers = [
   "Topic :: Database"
 ]
 
+[project.optional-dependencies]
+tests = [
```

```suggestion
dev = [
```
Maybe more broad due to pre-commit?